shims.h

```diff
@@ -93,222 +53,224 @@ typedef void* GO_SHA_CTX_PTR;
 // #if OPENSSL_VERSION_NUMBER < 0x10100000L
 // #include <openssl/bn.h>
 // #endif
```

Comment:
Now that we have two processors looking at this file, worth making it more explicit which processor cares about these lines (and that one of them does care at all!) by using a prefix or wrapping it somehow?

Reply:
Good point. I've added a comment to explain why this includes are here. It think this is enough.